packages/terra-show-hide/src/ShowHide.module.scss

```diff
@@ -14,4 +22,12 @@
   .end {
     margin-left: auto;
   }
+
+  .show-hide-content {
+    &:focus,
+    :focus {
+      outline: var(--terra-show-hide-focus-outline, 1px dashed);
```

Comment:
Isn't this different than our normal border style for focus?

Reply:
Yes, it was provided by @eawww